dev/io.openliberty.mcp.internal/src/io/openliberty/mcp/internal/tools/ToolManager.java

```diff
@@ -151,6 +152,8 @@ public interface ToolArguments extends RequestFeatureArguments {
 
         Map<String, Object> args();
 
+        RequestId id();
+
```

Comment:
Can we remove this change, and instead uncomment the relevant part of `FeatureManager.RequestFeatureArguments` please?

This will also mean renaming it to `requestId`, rather than `id` in `ToolArgumentsImpl`.